site/jekyll/guides/server-side-rendering.md

```diff
@@ -18,8 +18,21 @@ namespace MyApp
 	{
 		public static void Configure()
 		{
+			// For basic server-side rendering of React components
+			// add all necessary JavaScript files here. This includes
+			// all dependencies.
+			ReactSiteConfiguration.Configuration
+				.AddScript("~/Scripts/First.jsx")
+				.AddScript("~/Scripts/DependencyOfSecondComponent.jsx")
+				.AddScript("~/Scripts/SecondComponent.jsx");
+			
+			// This setup assumes the use of a build tool (Babel,
+			// Webpack, Gulp, Browserify, etc). Performance is improved
+			// when you disable ReactJS.NET's version of Babel and load
+			// pre-transpiled scripts. Example:
 			ReactSiteConfiguration.Configuration = new ReactSiteConfiguration()
-				.AddScript("~/Scripts/HelloWorld.jsx");
+				.SetLoadBabel(false)
```

Comment:
I don't think it makes sense to have both of these sections. I'd put them in two separate code blocks (eg. "here's how to add scripts normally" and "here's how to add scripts if you're using a build tool")